doc/api/esm.md

```diff
@@ -0,0 +1,85 @@
+# Modules
+
+<!--introduced_in=v9.x.x-->
+
+> Stability: 1 - Experimental
+
+<!--name=esm-->
+
+Node contains support for ES Modules based upon [the Node EP for ES Modules](https://github.com/nodejs/node-eps/blob/master/002-es-modules.md).
```

Comment:
Nit: move the links to an index at the end of the doc and use the abbreviated `[the Node EP for ES Modules][]` syntax in line. Also, lines should wrap at < 80 chars